infra/modules/service/events_jobs.tf

```diff
@@ -98,10 +86,85 @@ resource "aws_cloudwatch_event_target" "document_upload_jobs" {
     input_template = replace(replace(jsonencode({
       containerOverrides = [
         {
-          name    = local.container_name,
           command = each.value.task_command
         }
       ]
     }), "\\u003c", "<"), "\\u003e", ">")
   }
 }
```

Comment:
I think we can simplify this:

```
input_template = replace(replace(jsonencode({
  task_command = each.value.task_command
}), "\\u003c", "<"), "\\u003e", ">")
```

then below where we set the container overrides we can do

```
"Command.$" : "$.task_command"
```